solidity/contracts/ConnextVestingWallet.sol

```diff
@@ -8,86 +8,110 @@ import {Ownable2Step} from '@openzeppelin/contracts/access/Ownable2Step.sol';
   uint64 public constant ONE_MONTH = ONE_YEAR / 12;
+  uint64 public constant SEPT_05_2023 = 1_693_872_000;
+  uint64 public constant NEXT_TOKEN_LAUNCH = SEPT_05_2023;
   address public constant NEXT_TOKEN = 0xFE67A4450907459c3e1FFf623aA927dD4e28c67a; // Mainnet NEXT token
 
-  uint64 public constant VESTING_OFFSET = ONE_YEAR - ONE_MONTH;
-  uint64 public constant VESTING_DURATION = ONE_YEAR + ONE_MONTH;
-  uint64 public constant VESTING_CLIFF_DURATION = ONE_MONTH;
-
-  uint64 public initTimestamp;
-  uint256 public totalAmount;
+  /**
+   * NOTE:  The equivalent vesting schedule has a 13 months duration, with a 1 month cliff,
+   *        offsetted to start from `Sept 5th 2024 - 1 month`: At Sept 5th 2024 the cliff
+   *        is triggered unlocking 1/13 of the tokens, and then 1/13 of the tokens will
+   *        be linearly unlocked every month after that.
+   */
+  uint64 public constant VESTING_DURATION = ONE_YEAR + ONE_MONTH; //                  13 months duration
+  uint64 public constant VESTING_CLIFF_DURATION = ONE_MONTH; //                       1 month cliff
+  uint64 public constant VESTING_OFFSET = ONE_YEAR - ONE_MONTH; //                    11 months offset
+  uint64 public constant VESTING_START_DATE = NEXT_TOKEN_LAUNCH + VESTING_OFFSET; //  Sept 5th 2024 - 1 month
+
+  /**
+   * --- Settable Storage ---
+   */
+  uint256 public immutable TOTAL_AMOUNT;
 
   constructor(
-    uint64 _initTimestamp,
     address _beneficiary,
     uint256 _totalAmount
-  ) VestingWalletWithCliff(_beneficiary, _initTimestamp + VESTING_OFFSET, VESTING_DURATION, VESTING_CLIFF_DURATION) {
-    totalAmount = _totalAmount;
-    initTimestamp = _initTimestamp;
+  ) VestingWalletWithCliff(_beneficiary, VESTING_START_DATE, VESTING_DURATION, VESTING_CLIFF_DURATION) {
+    TOTAL_AMOUNT = _totalAmount;
   }
 
-  error NoVestingAgreement();
+  /**
+   * --- Errors ---
+   */
+  error NotAllowed();
   error ZeroAddress();
 
-  /// @inheritdoc VestingWallet
-  /// @dev This contract is only meant to vest NEXT tokens
+  /**
+   * --- Overrides ---
+   */
+
+  /**
+   * @inheritdoc  VestingWallet
+   * @notice      This contract is only meant to vest NEXT tokens
+   */
   function vestedAmount(uint64) public view virtual override returns (uint256 _amount) {
-    revert NoVestingAgreement();
+    return 0;
   }
 
-  /// @inheritdoc VestingWallet
-  /// @dev This contract is only meant to vest NEXT tokens
+  /**
+   * @inheritdoc  VestingWallet
+   * @notice      This contract is only meant to vest NEXT tokens
+   */
   function vestedAmount(address _token, uint64 _timestamp) public view virtual override returns (uint256 _amount) {
-    if (_token != NEXT_TOKEN) revert NoVestingAgreement();
-
-    return _vestingSchedule(totalAmount, _timestamp);
-  }
+    if (_token != NEXT_TOKEN) return 0;
 
-  /// @inheritdoc VestingWallet
-  /// @dev This contract is only meant to vest NEXT tokens
-  function releasable() public view virtual override returns (uint256 _amount) {
```

Comment:
`releasable()` already returns 0 (bc `vestedAmount()` returns 0), so no need to override.